### PR TITLE
0.7.0

### PR DESCRIPTION
**Pull request by ca333**
0.7.0 [Medusa]

changelog:

disable vSolutions size > 1 after HF
fix `-disablewallet` arg on getblocktemplate caused daemon crash
add zHTLC functionality 
update assetchains file
add S5 pubkeys and HF bits
update CI
bump version
general enhancements

### CODE REVIEW

**Comment by DeckerSU** [comment]
@ca333 it's final **0.7.0** version? I mean, will we include https://github.com/KomodoPlatform/komodo/pull/429 here as well? Also, what we will decide about versioning - https://github.com/KomodoPlatform/komodo/pull/431#discussion_r632383864 ?

**Comment by ca333** [comment]
> @ca333 it's final **0.7.0** version? I mean, will we include #429 here as well? Also, what we will decide about versioning - [#431 (comment)](https://github.com/KomodoPlatform/komodo/pull/431#discussion_r632383864) ?

thanks @DeckerSU - merged #429 and have the versioning (client and komodo version) synced on `0.7.0`

**Review by smk762 — APPROVED** [review]
pubkeys / consts etc ok

**Comment by Alrighttt** [comment]
Consensus changes are good. Assuming the configure related stuff and dep related things are good, I will approve.

File: .github/workflows/komodod_cd.yml [hunk 1]
```diff
@@ -109,11 +110,14 @@ jobs:
...
+          sudo apt-get remove php* msodbcsql17 mysql* powershell containernetworking-* containers* dotn*
           sudo apt-get update
           sudo apt-get upgrade -y
-          sudo apt-get install build-essential pkg-config libc6-dev m4 g++-multilib autoconf libtool libncurses-dev unzip git python zlib1g-dev wget bsdmainutils automake libboost-all-dev libssl-dev libprotobuf-dev protobuf-compiler libqrencode-dev libdb++-dev ntp ntpdate nano software-properties-common curl libevent-dev libcurl4-gnutls-dev cmake clang libsodium-dev -y
-          sudo apt-get install build-essential pkg-config libc6-dev m4 g++-multilib autoconf libtool ncurses-dev unzip git python python-zmq zlib1g-dev wget libcurl4-gnutls-dev bsdmainutils automake curl cmake mingw-w64
+          sudo apt-get install build-essential pkg-config libc6-dev m4 g++-multilib autoconf libtool \
+                               libncurses-dev unzip wget bsdmainutils automake libboost-all-dev libssl-dev \
+                               libprotobuf-dev protobuf-compiler libqrencode-dev libdb++-dev ntp ntpdate nano \
```
Comment (Alrighttt) [inline .github/workflows/komodod_cd.yml:118]: Unfamiliar with CI flow, but maybe we want to take a second look at these? For example, how could nano be a dep?

**Review by Alrighttt — APPROVED** [review]

File: .github/workflows/komodod_cd.yml (hunk 1 above)
Comment (SirSevenG) [reply .github/workflows/komodod_cd.yml:118]: I believe better cd workflow was in research branch, will recheck and PR updates later.
Also, we are not really using it as far as i'm aware. Still, no reason to keep it messy, yeah.

**Pull request merged**